CMakeLists.txt

```diff
@@ -170,35 +170,33 @@ IF(MSVC AND BUILD_SHARED_LIBS AND LOG4CXX_INSTALL_PDB)
   )
 ENDIF()
 
-if(UNIX)
-  # Support for pkg-config in consuming projects
-  set(prefix "${CMAKE_INSTALL_PREFIX}")
-  set(exec_prefix "${CMAKE_INSTALL_PREFIX}")
-  set(libdir "\${prefix}/${CMAKE_INSTALL_LIBDIR}")
-  set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
-  set(VERSION "${log4cxx_VERSION_MAJOR}.${log4cxx_VERSION_MINOR}.${log4cxx_VERSION_PATCH}")
-  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/liblog4cxx.pc.in"
-    "${CMAKE_CURRENT_BINARY_DIR}/lib${LOG4CXX_LIB_NAME}.pc"
-  )
+# Support for pkg-config in consuming projects
```

Comment:
Is `pkg-config` available on Windows?  I don't think we want to always install the .pc file.

Adding a new option to force the .pc installation feels better to me, I'm not sure if anybody else has any thoughts.

Reply:
Yes, It's available.
Does it hurt to install pc file?